modelgrid_aliases_nuopc.xml

```diff
@@ -222,140 +228,28 @@
-    <grid name="glc">gris20</grid>
-    <grid name="ocnice">tnx1v4</grid>
-    <mask>tnx1v4</mask>
-  </model_grid>
-  <model_grid alias="f09_g17_gris20" compset="_CISM" not_compset="_BLOM">
-    <grid name="atm">0.9x1.25</grid>
-    <grid name="lnd">0.9x1.25</grid>
-    <grid name="glc">gris20</grid>
-    <grid name="ocnice">gx1v7</grid>
-    <mask>gx1v7</mask>
-  </model_grid>
-  <model_grid alias="f09_tn14_ais8" compset="_CISM" not_compset="_POP|_MOM">
-    <grid name="atm">0.9x1.25</grid>
-    <grid name="lnd">0.9x1.25</grid>
-    <grid name="glc">ais8</grid>
-    <grid name="ocnice">tnx1v4</grid>
-    <mask>tnx1v4</mask>
-  </model_grid>
-  <model_grid alias="f09_tn14_ais8gris4" compset="_CISM" not_compset="_POP|_MOM">
-    <grid name="atm">0.9x1.25</grid>
-    <grid name="lnd">0.9x1.25</grid>
-    <grid name="glc">ais8:gris4</grid>
-    <grid name="ocnice">tnx1v4</grid>
-    <mask>tnx1v4</mask>
-  </model_grid>
-  <model_grid alias="f09_g17_ais8" compset="_CISM" not_compset="_BLOM">
-    <grid name="atm">0.9x1.25</grid>
-    <grid name="lnd">0.9x1.25</grid>
-    <grid name="glc">ais8</grid>
-    <grid name="ocnice">gx1v7</grid>
     <mask>gx1v7</mask>
   </model_grid>
-  <model_grid alias="f09_g17_ais8gris4" compset="_CISM" not_compset="_BLOM">
+  <model_grid alias="f09_f09_mtn14" compset="_DOCN" >
     <grid name="atm">0.9x1.25</grid>
     <grid name="lnd">0.9x1.25</grid>
-    <grid name="glc">ais8:gris4</grid>
-    <grid name="ocnice">gx1v7</grid>
-    <mask>gx1v7</mask>
-  </model_grid>
-
-  <model_grid alias="f19_tn14_wtn14" not_compset="_POP|_MOM">
-    <grid name="atm">1.9x2.5</grid>
-    <grid name="lnd">1.9x2.5</grid>
-    <grid name="ocnice">tnx1v4</grid>
-    <grid name="wav">wtnx1v4</grid>
-    <mask>tnx1v4</mask>
-  </model_grid>
-  <model_grid alias="f19_tn14_wtn14nw" not_compset="_POP|_MOM">
-    <grid name="atm">1.9x2.5</grid>
-    <grid name="lnd">1.9x2.5</grid>
-    <grid name="ocnice">tnx1v4</grid>
-    <grid name="wav">wtnx1v4nw</grid>
-    <mask>tnx1v4</mask>
-  </model_grid>
-
-  <model_grid alias="f19_tn14_w025" not_compset="_POP|_MOM">
-    <grid name="atm">1.9x2.5</grid>
-    <grid name="lnd">1.9x2.5</grid>
-    <grid name="ocnice">tnx1v4</grid>
-    <grid name="wav">w025</grid>
-    <mask>tnx1v4</mask>
-  </model_grid>
-
-  <model_grid alias="f19_tn14_wtnx1v4p100k" not_compset="_POP|_MOM">
-    <grid name="atm">1.9x2.5</grid>
-    <grid name="lnd">1.9x2.5</grid>
-    <grid name="ocnice">tnx1v4</grid>
-    <grid name="wav">wise.tnx1v4p.100k</grid>
+    <grid name="ocnice">0.9x1.25</grid>
     <mask>tnx1v4</mask>
   </model_grid>
-
-  <model_grid alias="f19_g17_wg17" compset="_CAM">
-    <grid name="atm">1.9x2.5</grid>
-    <grid name="lnd">1.9x2.5</grid>
-    <grid name="ocnice">gx1v7</grid>
-    <grid name="wav">gx1v7</grid>
-    <mask>gx1v7</mask>
-  </model_grid>
-
-  <model_grid alias="f09_tn14_ais8" compset="_CISM" not_compset="_POP|_MOM">
+  <model_grid alias="f09_tn14" not_compset="_POP|_MOM" >
```

Comment:
I don't think this needs to be changed for this PR but would it be more future-proof to replace this pattern with:
```suggestion
  <model_grid alias="f09_tn14" compset="_BLOM|_DOCN" >
```
?